backend/src/bundles/users/enums/enums.ts

```diff
@@ -1,3 +1,9 @@
 export { UserAttributes } from './user-attributes.enum.js';
 export { UserDetailsAttributes } from './user-details-attributes.enum.js';
 export { Gender, UsersApiPath } from 'shared';
+export {
+    AuthApiPath,
+    HttpCode,
+    HttpError,
```

Comment:
You have HttpCode, HttpError in **backend/src/common/http/http.ts**, I guess you don't need this export here